app/views/dashboards/_actions.html.erb

```diff
@@ -1,19 +1,53 @@
-<div class="actions">
-  <a class="action <%= "active" if params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) %>" href="/dashboard">
-    <span>POSTS</span>
-    <span>(<%= @user.articles_count %>)</span>
+<nav class="hidden m:block">
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) %>" href="/dashboard">
+    Posts
+    <span class="crayons-indicator"><%= @user.articles_count %></span>
   </a>
-  <a class="action <%= "active" if params[:action] == "followers" %>" href="/dashboard/user_followers">
-    <span>FOLLOWERS</span>
-    <span>(<%= @user.followers_count %>)</span>
+
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "followers" %>" href="/dashboard/user_followers">
+    Followers
+    <span class="crayons-indicator"><%= @user.followers_count %></span>
   </a>
-  <a class="action <%= "active" if params[:action] == "following" || params[:action].include?("following") %>" href="/dashboard/following">
-    <span>FOLLOWING</span>
-    <span>(<%= @user.following_users_count + @user.following_tags_count + @user.following_organizations_count + @user.following_podcasts_count %>)</span>
+
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "following_tags" %>" href="/dashboard/following_tags">
+    Following tags
+    <span class="crayons-indicator"><%= @user.following_tags_count %></span>
   </a>
-  <% if @user.has_role? :pro %>
-    <a class="action" href="/dashboard/pro">
-      <span>PRO ANALYTICS</span>
+
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "following_users" %>" href="/dashboard/following_users">
+    Following users
+    <span class="crayons-indicator"><%= @user.following_users_count %></span>
+  </a>
+
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "following_organizations" %>" href="/dashboard/following_organizations">
+    Following orgnizations
+    <span class="crayons-indicator"><%= @user.following_organizations_count %></span>
+  </a>
+
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "following_podcasts" %>" href="/dashboard/following_podcasts">
+    Following podcasts
+    <span class="crayons-indicator"><%= @user.following_podcasts_count %></span>
+  </a>
+
+  <a class="crayons-link crayons-link--block" href="/listings/dashboard" data-no-instant>
+    Listings
+    <%= inline_svg_tag("external-link.svg", aria: true, class: "crayons-icon", title: "Moderation") %>
+  </a>
```

Comment:
given the external link icon I'd expect this to open in a new tab, no?

Reply:
It's temporary - in this case it means it will take you "outside" of your dashboard since "listings dashboard" is not yet redesigned... We will eventually combine the two and this icon will be gone. For now, I'm gonna say it's ok.